lib/Extended_ast.ml

```diff
@@ -111,32 +115,63 @@ module Parse = struct
     | Core_type -> Parse.core_type lexbuf
     | Module_type -> Parse.module_type lexbuf
     | Expression -> Parse.expression lexbuf
+    | Repl_file ->
+        let str = String.strip (Bytes.to_string lexbuf.lex_buffer) in
+        let phrases = Astring.String.cuts ~sep:"\n# " str in
+        let phrases =
+          List.mapi phrases ~f:(fun i p ->
+              let p = String.strip p in
+              match i with
+              | 0 ->
+                  if String.is_prefix p ~prefix:"# " then
```

Comment:
`String.drop_prefix` should do what you want

Reply:
sorry, I meant `String.chop_prefix `, combining both `is_prefix` and `drop_prefix`

Reply:
done